netlogo-gui/src/main/app/App.scala

```diff
@@ -847,6 +832,13 @@ class App extends
     showAboutWindow()
   }
 
+  /**
+   * This is called reflectively by the mac app wrapper.
+   */
+  def handleShowPreferences(): Unit = {
+    showPreferencesDialog()
```

Comment:
How can it break mac-app in a way that would not break if called by handleShowAbout?